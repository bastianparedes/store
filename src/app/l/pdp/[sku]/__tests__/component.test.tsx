import React from 'react';

import { render } from '@testing-library/react';

import Component from '../component';

const product = {
  description: 'description',
  name: 'name',
  ownerEmail: 'test@test.cl',
  price: 1000,
  quantity: 10,
  sku: 1
};

it('should render', () => {
  const { container } = render(<Component product={product} />);
  expect(container).toMatchSnapshot();
});
