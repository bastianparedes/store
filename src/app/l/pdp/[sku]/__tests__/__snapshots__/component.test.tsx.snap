// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="product"
    >
      <div
        class="imageContainer"
      >
        <img
          alt="name"
          src="/api/image/1"
        />
      </div>
      <div
        class="productInfo"
      >
        <h1>
          name
        </h1>
        <p>
          description
        </p>
        <p>
          SKU: 
          1
        </p>
        <p>
          Cantidad disponible: 
          10
        </p>
        <p>
          Precio: $
          1000
        </p>
      </div>
    </div>
  </div>
</div>
`;
